osm2gtfs/creators/fenix/trips_creator_fenix.py

```diff
@@ -230,14 +240,17 @@ def get_exception_service_period(self, feed, date, day):
     @staticmethod
     def match_first_stops(route, sim_stops):
         # get the first stop of the route
-        stop = route.get_first_stop()
+        stop = route.stops[0]
 
         # normalize its name
         stop.name = TripsCreatorFenix.normalize_stop_name(stop.name)
 
         # get first stop from relation 'from' tag
-        alt_stop_name = route.get_first_alt_stop()
-        alt_stop_name = TripsCreatorFenix.normalize_stop_name(alt_stop_name.encode('utf-8'))
+        if 'from' in route.tags:
+            alt_stop_name = route.tags['from']
+        else:
+            alt_stop_name = ""
+        alt_stop_name = TripsCreatorFenix.normalize_stop_name(alt_stop_name)
```

Comment:
Why did you add `TripsCreatorFenix` before the method call? Are we not in this class at the moment?

Reply:
Yes, you are right. Changed to `self`.